openmrs/omod/src/main/java/org/openmrs/projectbuendia/servlet/DataExportServlet.java

```diff
@@ -89,11 +90,49 @@
     private static final int COLUMNS_PER_OBS = 3;
     private static final ClientConceptNamer NAMER = new ClientConceptNamer(Locale.ENGLISH);
 
+    public static final int DEFAULT_INTERVAL = 30;
```

Comment:
please always include units in the name for time-related constants, e.g. `DEFAULT_INTERVAL_MINS`.